cafebabel/templates/articles/edit.html

```diff
@@ -51,7 +51,7 @@ <h1 class="edit {% if article.id %}active{% endif %}">
     <label for=summary>Lead paragraph</label>
     <textarea id=summary name=summary required>{{ article.summary or '' }}</textarea>
   </div>
-  <div class=textarea>
+  <div class="textarea bodytext">
```

Comment:
`markdowntext` is probably more pertinent given that we will probably use it for tags and so on.